src/System.Text.JsonLab.Serialization/System/Text/Json/Serialization/Policies/DateConverter.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers.Text;
+
+namespace System.Text.Json.Serialization.Policies
+{
+    internal class DateConverter : IUtf8ValueConverter<DateTime>
```

Comment:
Would this still be needed after https://github.com/dotnet/corefx/issues/34690?

Reply:
Perhaps if someone has a special encoding of date (there are many)